lib/syspro/api_operations/transaction.rb

```diff
@@ -0,0 +1,35 @@
+# frozen_string_literal: true
+
+module Syspro
+  module ApiOperations
+    module Transaction
+      module ClassMethods
+        def post(params)
+          request(:get, '/Transaction/Post', params)
+        end
+
+        private
+
+        def warn_on_opts_in_params(params)
+          Util::OPTS_USER_SPECIFIED.each do |opt|
+            if params.key?(opt)
+              warn("WARNING: #{opt} should be in opts instead of params.")
+            end
+          end
```

Comment:
Nothing wrong with this code, just wanted to point out a functional way of doing the same thing:
```
Util::OPTS_USER_SPECIFIED.select { |opt| params.key?(opt) }.each do |opt|
  warn(...)
end
```

Reply:
Cool! Thanks